h2o-algos/src/main/java/hex/schemas/AnovaGLMV3.java

```diff
@@ -0,0 +1,161 @@
+package hex.schemas;
+
+import hex.anovaglm.AnovaGLM;
+import hex.anovaglm.AnovaGLMModel;
+import hex.glm.GLMModel;
+import water.api.API;
+import water.api.schemas3.KeyV3;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+public class AnovaGLMV3 extends ModelBuilderSchema<AnovaGLM, AnovaGLMV3, AnovaGLMV3.AnovaGLMParametersV3> {
+
+  public static final class AnovaGLMParametersV3 extends ModelParametersSchemaV3<AnovaGLMModel.AnovaGLMParameters,
+          AnovaGLMParametersV3> {
+    public static final String[] fields = new String[] {
+            "model_id",
+            "training_frame",
+            "seed",
+            "response_column",
+            "ignored_columns",
+            "ignore_const_cols",
+            "score_each_iteration",
+            "offset_column",
+            "weights_column",
+            "family",
+            "tweedie_variance_power",
+            "tweedie_link_power",
+            "theta", // equals to 1/r and should be > 0 and <=1, used by negative binomial
+            "solver",
+            "missing_values_handling",
+            "plug_values",
+            "compute_p_values",
+            "standardize",
+            "non_negative",
+            "max_iterations",
+            "link",
+            "prior",
+            "alpha",
+            "lambda",
+            "lambda_search",
+            "stopping_rounds",
+            "stopping_metric",
+            "early_stopping",
+            "stopping_tolerance",
+            // dead unused args forced here by backwards compatibility, remove in V4
```

Comment:
it looks like we are introducing unused parameters to a new algo

Reply:
Yes, they are necessary though to specify the actions needed to perform.  They are found in R as well.

Reply:
why are they marked as "dead"?

Reply:
I copied them over from GLMV3.java.  I removed the comments.